### DISCUSSION TITLE
pcl2会被杀

### DISCUSSION THREAD

**Discussion by minecraftlin** [Meloong-Git/PCL#1588] [提问] [answered]
我的pcl2好几天没用结果被defender杀了........
希望可以解决下这个问题

**Comment by WForst-Breeze** [comment, marked as answer]
快照版本 PCL2 会被误报，请将 PCL2 添加至相关杀软白名单中，此处以 Windows Defender 为例：[将排除项添加到 Windows 安全中心](https://support.microsoft.com/zh-cn/windows/%E5%B0%86%E6%8E%92%E9%99%A4%E9%A1%B9%E6%B7%BB%E5%8A%A0%E5%88%B0-windows-%E5%AE%89%E5%85%A8%E4%B8%AD%E5%BF%83-811816c0-4dfd-af4a-47e4-c301afe13b26)
